about.html

```diff
@@ -0,0 +1,128 @@
+    <link rel="stylesheet" href="about.css">
+    <title>BEAUTY SUMMIT</title>
+</head>
+<body class="body">
+    <section class="beauty-summit">
+        <div class="header-links">
+            <ul class="top-header-links">
+                <li><a href="#" class="facebook"><img src="Logo/facebook.png"></a></li>
+                <li><a href="#" class="twitter"><img src="Logo/twitter.png"></a></li>
+                <li><a href="#" class="header-items">English</a></li>
+                <li><a href="#" class="header-items">MyPage</a></li>
+                <li><a href="#" class="header-items">Logout</a></li>
+            </ul>
+        </div>
+        <header class="header">
+            <a href="home.html" class="myLogo"><img src="Logo/logo2.jpg"></a>
+
+            <nav class="nav">
+                <a href="" class="menu"></a>
+                <ul class="nav-links-desktop">
+                    <li class="nav-menu"><a href="about.html" class="nav-links-items">About</a></li>
+                    <li class="nav-menu"><a href="#" class="nav-links-items">Program</a></li>
+                    <li class="nav-menu"><a href="#" class="nav-links-items">Join</a></li>
+                    <li class="nav-menu"><a href="#" class="nav-links-items">Sponsor</a></li>
+                    <li class="nav-menu"><a href="#" class="nav-links-items">News</a></li>
+                    <li class="nav-menu-bt"><a href="#" class="nav-links-items-bt">CC campaign</a></li>
+
+                </ul>
+                <div id="mob-nav" class="mob-nav">
+                    <div class="menu-btn">
+                        <div class="hamburger">
+                          <a href="#"
+                            ><img
+                              src="Logo/mobile_menu 1.png"
+                              alt="hamburger"
+                              class="hamBtn"
+                          /></a>
+                        </div>
+                      </div>
+                      <div class="mob-menu">
+                        <ul class="menu-list">
+                          <a href="about.html"><li class="menu-items">About</li></a>
+                          <a href="#"><li class="menu-items">Program</li></a>
+                          <a href="#"><li class="menu-items">Feature</li></a>
+                          <a href="#"><li class="menu-items">Sponsor</li></a>
+                          <a href="#"><li class="menu-items">News</li></a>
+                          <a href="#"><li class="menu-items">CC Campaign</li></a>
+                          <a href="#"><li class="menu-items"></li></a>
+                        </ul>
+                        <div class="closeBtn">&times;</div>
+                      </div>
+
+                </div>
+            </nav>
+        </header>
+        <div class="homepage">
+            <h2 class="homepage-h2">"Hello! Sharing World"</h2>
+            <h1 class="homepage-h1">BEAUTY TECH AND INNOVATION SUMMIT 2023</h1>
```

Comment:
- To comply with web development best practices, I think you can not have more than a `h1` in one document. This will allow crawling and indexing engines not to have issues understanding what your page is about. 
- - [ ] Kindly use another `hx` tag for this purpose :wink:

Reply:
- [ ] I am sorry, it looks like I didn't save an in-line comment before the general comment. In your `about.html`, I have provided a view of how it looks and how to improve it so that you get it approved in your next attempt.

![image](https://github.com/Cheptoo21/Capstone-Project-1/assets/99715640/8677f09f-79f9-43d3-962f-51dde7367036)

- [ ] Kindly note the previous similar suggestions on the home.html - header section, the closeness of the menu button to the contents on the page, and now, to add, the subtitles are not having the same font-sizes - please check that :clap: